### ISSUE TITLE
Unknown motherboard, contact us on github to resolve this.

### ISSUE THREAD

**Issue by kiven7e** [MrPotatoBobx/FetchCord#151]
# Debug info

`Symlinking XDG_RUNTIME_DIR path for Flatpak Discord.
kiven@linux 
----------- 
OS: Debian GNU/Linux 11 (bullseye) x86_64 
Kernel: 5.10.0-6-amd64 
Uptime: 2 hours, 19 mins 
Packages: 2470 (dpkg), 25 (brew), 12 (flatpak), 12 (snap) 
Shell: bash 5.1.4 
Resolution: 1920x1080, 1680x1050 
DE: GNOME 3.38.4 
WM: Mutter 
WM Theme: Adwaita 
Theme: Yaru-Deepblue-dark [GTK2/3] 
Icons: Yaru-Deepblue [GTK2/3] 
Terminal: gnome-terminal 
CPU: Intel i5-10400 (12) @ 4.300GHz 
GPU: AMD ATI Radeon HD 7970/8970 / R9 280X 
GPU: AMD ATI Radeon HD 7970/8970 / R9 280X 
Memory: 5287MiB / 15905MiB 
Disk (/): 277G / 457G (64%)
[KeyError]: 'Font:'[KeyError]: 'Font:'[KeyError]: 'Host:'
----run_rpc----

uptime in epoch: 1622215173.0
cpuid: 740490017218232392
cpuappid: 741099939198926920
termappid: 741328861115056160
[KeyError]: 'Host:'
2470 (dpkg), 25 (brew), 12 (flatpak), 12 (snap) 
----out.py----

----DE/WM----
deid: GNOME
wmid: Mutter
wmline item 0: Mutter 

----TERMINAL----

[KeyError]: 'Font:'
fontline: Font: N/A
termid: 741328861115056160
termline item 0: gnome-terminal 
themeline: Adwaita 
Yaru-Deepblue-dark [GTK2/3] 
[KeyError]: 'Host:'

----GPU INFO----

gpuinfo: bash: line 1: glxinfo: command not found
bash: line 1: glxinfo: command not found
gpuvendor: amd

----CPU INFO----

cpuvendor: intel
cpumodel: Intel i5
cpuinfo: Intel i5-10400 (12) @ 4.300GHz
cpuline item 0: Intel i5-10400 (12) @ 4.300GHz
memline: 5287MiB / 15905MiB 

----OS INFO----

sysosline: Debian GNU/Linux 11 (bullseye) x86_64 
sysosid: 740490017218232392
diskline: /): 277G / 457G (64%)
packagesline item 0: 2470 (dpkg), 25 (brew), 12 (flatpak), 12 (snap) 

----testing.py----
----DE/WM----

deid: GNOME
wmid: Mutter

----TERMINAL/SHELL----

termid: 741328861115056160
shellid: bash

----HOST INFO----

[KeyError]: 'Host:'
[KeyError]: 'Host:'
hostid: 742887089179197462

----GPU INFO----

gpuvendor: amd

----CPU INFO----

cpumodel: Intel i5

('DEFAULT', <Section: DEFAULT>)
('cycle_0', <Section: cycle_0>)
('cycle_1', <Section: cycle_1>)
('cycle_2', <Section: cycle_2>)
('cycle_3', <Section: cycle_3>)
[KeyError]: 'Host:'
Unknown motherboard, contact us on github to resolve this.
{}
cycle0 not in : True
try_connect(key="cycle0") on <pypresence.presence.Presence object at 0x7f5c4bae55b0>
cycle 0
try_update(key="cycle0") on <pypresence.presence.Presence object at 0x7f5c4bae55b0>
appid: 740490017218232392
{'cycle0': <pypresence.presence.Presence object at 0x7f5c4bae55b0>}
cycle1 not in : True
try_connect(key="cycle1") on <pypresence.presence.Presence object at 0x7f5c4bae5670>
cycle 1
try_update(key="cycle1") on <pypresence.presence.Presence object at 0x7f5c4bae5670>
appid: 741099939198926920

`
# Operating system & way of installation

`Debian Bullseye`
`sudo apt install neofetch python3-pip`
`pip3 install fetchcord`

# Error/issue
`kiven@bullseye:~$ fetchcord
Symlinking XDG_RUNTIME_DIR path for Flatpak Discord.
[KeyError]: 'Font:'[KeyError]: 'Font:'[KeyError]: 'Host:'
('DEFAULT', <Section: DEFAULT>)
('cycle_0', <Section: cycle_0>)
('cycle_1', <Section: cycle_1>)
('cycle_2', <Section: cycle_2>)
('cycle_3', <Section: cycle_3>)
[KeyError]: 'Host:'
Unknown motherboard, contact us on github to resolve this.
Symlinking XDG_RUNTIME_DIR path for Flatpak Discord.
[KeyError]: 'Host:'
Symlinking XDG_RUNTIME_DIR path for Flatpak Discord.
[KeyError]: 'Host:'
Symlinking XDG_RUNTIME_DIR path for Flatpak Discord.
[KeyError]: 'Host:'
Symlinking XDG_RUNTIME_DIR path for Flatpak Discord.
`

# Other notes(optional)

i have an asus prime z490-a motherboard

**Comment by BlivionIaG** [comment]
Hello, what is your neofetch Host line ? I can't add your motherboard without it (or model reported by neofetch), it might be some config issue.

**Comment by ebbit1q** [comment]
hi, I have this same issue, it is caused by neofetch being unable to fetch the Host because it is simply not set

neofetch fetches the host by checking the value of `/sys/devices/virtual/dmi/id/product_name` and `/sys/devices/virtual/dmi/id/product_version`, see here https://github.com/dylanaraps/neofetch/blob/master/neofetch#L1238

for me these values are set to `To be filled by O.E.M.` because my machine was not assembled by an OEM, neofetch discards this value intentionally because it's basically useless, preventing the Host: line from appearing

note that the get_model function does not fetch the motherboard properties, which are stored in `/sys/devices/virtual/dmi/id/board_vendor` and `/sys/devices/virtual/dmi/id/board_name` instead, these values are actually unused by neofetch

**Comment by BlivionIaG** [comment]
I see, well i'll have to add this for linux then.

**Comment by ebbit1q** [comment]
does neofetch display the motherboard properties on other oses?

I think simply skipping over the pc oem and just not displaying it is an acceptable compromise, alternatively it can just say that the pc is custom built if the Host line is missing

**Comment by MrPotatoBobx** [comment]
> does neofetch display the motherboard properties on other oses?
> 
> I think simply skipping over the pc oem and just not displaying it is an acceptable compromise, alternatively it can just say that the pc is custom built if the Host line is missing

I agree with this idea, this repo is flooded with motherboard issues

**Comment by Lukas-Batema** [comment]
> does neofetch display the motherboard properties on other oses?
> 
> I think simply skipping over the pc oem and just not displaying it is an acceptable compromise, alternatively it can just say that the pc is custom built if the Host line is missing

I will start work on the latter idea :)
